hack/create.sh

```diff
@@ -48,11 +48,15 @@ mkdir -p _data/_out
 
 az group create -n $RESOURCEGROUP -l eastus --tags now=$(date +%s) >/dev/null
 
-if [[ -z "$AZURE_CLIENT_ID" ]]; then
-    set +x
+set +x
+# app-update - BYO AAD object (supported, production flow)
+# app-create - create one for you using AZURE_CLIENT_ID (unsuported, for tests only)
+if [[ -z "$AZURE_AAD_CLIENT_ID" ]]; then
     . <(hack/aad.sh app-create openshift.$RESOURCEGROUP.$DNS_DOMAIN $RESOURCEGROUP)
-    set -x
+else
```

Comment:
If AAD ID is defined, means we are bringing our own WebAPI App and we will update it for this cluster run.